src/widgets/splits/Split.cpp

```diff
@@ -1110,6 +1137,375 @@ void Split::showViewerList()
+                loadingLabel->hide();
+                performListSearch();
+            },
+            [chattersList, formatListItemText](auto error, auto message) {
+                auto errorMessage = formatChattersError(error, message);
+                chattersList->addItem(formatListItemText(errorMessage));
+            });
+    };
+
+    QObject::connect(searchBar, &QLineEdit::textEdited, this,
+                     performListSearch);
+
+    // Add broadcaster
+    if (channel->isBroadcaster() || channel->hasModRights())
+    {
+        addLabel("Broadcaster");
+        chattersList->addItem(channel->getName());
+        chattersList->addItem(new QListWidgetItem());
+    }
+
+    // Only broadcaster can get vips, mods can get viewers
+    if (channel->isBroadcaster())
+    {
+        auto helixApi = getHelix();
+
+        // Add moderators
+        helixApi->getModerators(
+            twitchChannel->roomId(), 1000,
+            [=](auto modList) {
+                std::vector<QString> modVector;
+                for (auto mod : modList)
+                {
+                    modVector.push_back(mod.userName);
+                }
+
+                // Add vips
+                helixApi->getChannelVIPs(
+                    twitchChannel->roomId(),
+                    [=](auto vips) {
+                        std::vector<QString> vipVector;
+                        for (auto vip : vips)
+                        {
+                            vipVector.emplace_back(vip.userName);
+                        }
+
+                        // Add chatters
+                        loadChatters(modVector, vipVector);
+                    },
+                    [chattersList, formatListItemText](auto error,
+                                                       auto message) {
+                        auto errorMessage = formatVIPListError(error, message);
+                        chattersList->addItem(formatListItemText(errorMessage));
+                    });
+            },
+            [this, chattersList, formatListItemText](auto error, auto message) {
+                auto errorMessage = formatModsError(error, message);
+                chattersList->addItem(formatListItemText(errorMessage));
+            });
+    }
+    else if (channel->hasModRights())
+    {
+        std::vector<QString> modList;
+        modList.push_back("Moderators cannot check who is a moderator");
+        std::vector<QString> vipList;
+        vipList.push_back("Moderators cannot check who is a vip");
+        loadChatters(modList, vipList);
+    }
+    else
+    {
+        chattersList->addItem(
+            formatListItemText("Due to Twitch restrictions, this feature is "
+                               "only available for moderators."));
+        chattersList->addItem(
+            formatListItemText("If you would like to see the Viewer list, you "
+                               "must use the Twitch website."));
+        loadingLabel->hide();
+    }
```

Comment:
```suggestion
        chattersList->addItem(
            formatListItemText("Due to Twitch restrictions, this feature is "
                               "only \navailable for moderators."));
        chattersList->addItem(
            formatListItemText("If you would like to see the Viewer list, you "
                               "must \nuse the Twitch website."));
        loadingLabel->hide();
    }
```

I want to force a newline here as it would be better for the default window height.
before:
![image](https://user-images.githubusercontent.com/41973452/200187374-5714c4f4-c0d4-4ad4-bd91-867c43d576f2.png)
after:
![image](https://user-images.githubusercontent.com/41973452/200187453-d1598bd6-2f75-4e6e-819b-cc11a3c0eca9.png)



It looks meh no matter what variation if you have Chatterino full-screened
https://user-images.githubusercontent.com/41973452/200187408-6b7b7cb0-8b40-4856-a70b-8505120884ce.png
https://user-images.githubusercontent.com/41973452/200187482-9dd79d95-e53f-4a4b-a25f-286da77d11ff.png